validation/pipeline/qase/qase.go

```diff
@@ -0,0 +1,174 @@
+package qase
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/antihax/optional"
+	"github.com/sirupsen/logrus"
+	qase "go.qase.io/client"
+)
+
+type TestCaseSteps struct {
+	Action         string `json:"action,omitempty"`
+	ExpectedResult string `json:"expected_result,omitempty"`
+	InputData      string `json:"input_data,omitempty"`
+	Position       int32  `json:"position,omitempty"`
+}
+
+type TestCase struct {
+	Description string          `json:"description,omitempty"`
+	Title       string          `json:"title"`
+	SuiteId     int64           `json:"suite_id,omitempty"`
+	Automation  int32           `json:"automation,omitempty"`
+	Steps       []TestCaseSteps `json:"steps,omitempty"`
+}
+
+var (
+	qaseToken = os.Getenv(QaseTokenEnvVar)
+	Client    *qase.APIClient
```

Comment:
typically, we don't use var for clients. Instead, you can make the client part of a struct and define all its functions accordingly (similar to the structure of test files) or just pass the client along as a parameter to each function.

i.e. 
```
type QaseService struct {
  Client *qase.Client
}

func (q *QaseService) Setup {
...
}

...
func (q *QaseService) GetTestCase() () {
...}
```